README.md

```diff
@@ -20,6 +20,16 @@ Please use the [ISLE Documentation](https://islandora-collaboration-group.github
 * Time required < 30 minutes.
 * **Windows Users**: Please open the .env and uncomment `COMPOSE_CONVERT_WINDOWS_PATHS=1`
 
+## **AS OF THE ISLE v.1.4.0 release in November 2019 (MANDATORY)**
+
+Due to removal of older files and folders and git changes in the ISLE git repository as of `November 28 2019`, please review the following:
+
+* If you are a **new** ISLE user and are cloning or forking this project for the first time after `November 28, 2019`, then **NO** you don't need to pay attention to this notice and can move on to the Quick Start section as needed.
```

Comment:
Instead of "then **NO** you don't need to pay attention to this notice and can move on to the Quick Start section as needed." I'd suggest "then **you can ignore this section and move on to the Quick Start section.**"